lib/Common/ConfigFlagsList.h

```diff
@@ -1430,6 +1431,7 @@ FLAGNR(Boolean, RecyclerVerifyMark    , "verify concurrent gc", false)
 #endif
 FLAGR (Number,  LowMemoryCap          , "Memory cap indicating a low-memory process", DEFAULT_CONFIG_LowMemoryCap)
 FLAGNR(Number,  NewPagesCapDuringBGSweeping, "New pages count allowed to be allocated during background sweeping", DEFAULT_CONFIG_NewPagesCapDuringBGSweeping)
+FLAGR (Number,  AllocPolicyLimit      , "Memory allocation policy limit in MB (default: -1).", DEFAULT_CONFIG_AllocationPolicyLimit)
```

Comment:
>default: -1 [](start = 79, length = 11)

nit: should be better to word this as 
(default: -1, which means no allocation policy limit)
?